src/components/LibraryStageItem.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from "react";
+
+export interface LibraryStageItemProps {
+  label: string;
+  value: string;
+}
+export default class LibraryStageItem extends React.Component<LibraryStageItemProps, void> {
+  render(): JSX.Element {
+    let colors = {
+      "testing": "warning",
+      "production": "success",
+      "cancelled": "danger"
+    };
+
+    return(
+      <fieldset className="form-group well">
+        <legend>
+          <span>{this.props.label}</span>
+          <label className={`label label-${colors[this.props.value]}`}>{this.props.value}</label>
```

Comment:
This may be too long but something like `current library stage: ` would be better.